samza-core/src/main/java/org/apache/samza/processor/StreamProcessor.java

```diff
@@ -108,17 +108,6 @@ public StreamProcessor(int processorId, Config config, Map<String, MetricsReport
     this(processorId, config, customMetricsReporters, (Object) streamTaskFactory);
   }
 
-  /**
-   * Same as {@link #StreamProcessor(int, Config, Map, AsyncStreamTaskFactory)}, except task instances are created
-   * using the "task.class" configuration instead of a task factory.
-   * @param processorId - this processor Id
-   * @param config - config
-   * @param customMetricsReporters metrics
-   */
```

Comment:
Nvm. I checked your code changes.. Looks like you are removing it. I like it. It makes the stack more component-friendly and clean. Thanks!